index.html

```diff
@@ -1975,42 +2069,46 @@
+        A <a>match pattern</a> is used primarily for matching <a>IANA media type</a>s,
+        for instance the pattern <code>'application/*+json'</code> matches
         <code>'application/calendar+json'</code>, but does not match
-        <code>'application/json'</code>. The match pattern
-        <code>'*/*json'</code> matches both.
+        <code>'application/json'</code>. The pattern
+        <code>'*/*json'</code>, on the other hand, matches both.
       </p>
     </section>
     <section id="URL_pattern"> <h3>URL patterns</h3>
       <p>
-        A URL pattern is a URL which can contain <code>'*'</code>
-        characters. Each URL pattern is in the form of
-        <ul>
-          <li><em>scheme</em>— for example, 'https' or '*'</li>
-          <li><em>host</em> — for example, 'www.w3.org' or '*.w3.org'</li>
-          <li><em>path</em> — for example, '/*', '/foo*', or '/foo/bar'.</li>
-        </ul>
-      </p>
-      <p>
-        The syntax is the following:
+        A <a>URL pattern</a> is a URL which may contain the <code>'*'</code>
+        wildcard character.
+        It consists of a <a data-lt="star scheme">scheme</a>, a
+        <a data-lt="star host">host</a> and (optionally) a
+        <a data-lt="star path">path</a>, defined by the following ABNF:
         <pre>
-          <em>&lt;url-pattern&gt;</em> := <em>&lt;scheme&gt;</em>://<em>&lt;host&gt;</em><em>&lt;path&gt;</em>
-          <em>&lt;scheme&gt;</em> := '*' | 'https'
-          <em>&lt;host&gt;</em> := '*' | '*.' <em>&lt;any char except '/' and '*'&gt;</em>+
-          <em>&lt;path&gt;</em> := '/' <em>&lt;any chars&gt;</em>
+          <dfn data-lt="url pattern">url-pattern</dfn> = <a data-lt="star scheme">scheme</a> <code>"://"</code> <a data-lt="star host">host</a> <a data-lt="star path">path</a>
+          <dfn data-lt="star scheme">scheme</dfn>      = <code>"*"</code> | <code>"https"</code>
+          <dfn data-lt="star host">host</dfn>        = <code>"/"</code> &lt;any char&gt;
+          <dfn data-lt="star path">path</dfn>        = <code>"*"</code> | <code>"*."</code> &lt;any char except <code>"/"</code> and <code>"*"</code>&gt;
         </pre>
       </p>
-      <p>
-        Note that <code>'*'</code> has slightly different meaning depending on
-        whether it is in the <i>scheme</i>, <i>host</i> or <i>path</i> part.
+      <p class="note">
+        The <code>'*'</code> character has slightly different meaning depending
+        on whether it is in the <i>scheme</i>, <i>host</i> or <i>path</i> part.
         In a <i>scheme</i> <code>'*'</code> will match only the allowed
         <code>'https'</code> scheme.
         If the <i>host</i> is <code>'*'</code>, then it matches any host. If the
         <i>host</i> is e.g. <code>'*.myhost'</code>, then it will match the
-        specified host or any of its subdomains.
+        specified host or any of its subdomains. In the <i>host</i>
+        part, <code>'*'</code> MUST be the first character, so it can only be
```

Comment:
You can't use MUST for user input. Instead, say how it matches or fails to match if the * is somewhere other than the first character.